modules/draco/src/lib/draco-module-loader.ts

```diff
@@ -59,14 +63,28 @@ async function loadDracoDecoder(options) {
   let wasmBinary;
   switch (options.draco && options.draco.decoderType) {
     case 'js':
-      DracoDecoderModule = await loadLibrary(DRACO_JS_DECODER_URL, 'draco', options);
+      DracoDecoderModule = await loadLibrary(
+        (options.modules?.[DRACO_JS_DECODER_NAME] && DRACO_JS_DECODER_NAME) || DRACO_JS_DECODER_URL,
+        'draco',
+        options
+      );
       break;
 
     case 'wasm':
     default:
       [DracoDecoderModule, wasmBinary] = await Promise.all([
-        await loadLibrary(DRACO_WASM_WRAPPER_URL, 'draco', options),
-        await loadLibrary(DRACO_WASM_DECODER_URL, 'draco', options)
+        await loadLibrary(
+          (options.modules?.[DRACO_WASM_WRAPPER_NAME] && DRACO_WASM_WRAPPER_NAME) ||
```

Comment:
I am not convinced this is a good interface for end users:
1. It requires them to know the name of the decoder JS and WASM files.
2. It doesn't make it easy to configure these overrides together with other overrides.

I will admit that the system is not in the best of shape even before this PR, so I don't want to force this PR to clean everything up. That said, if we think about what a good solution would involve.
- I think one key use case is that the developer wants to make sure that all dependencies run locally. We should make it as easy as possible for the developer to copy all dependencies to a local directory and load them from there.
- The other use case is that someone just wants to override a single dynamically loaded component, and we should make that as easy as possible.